.github/workflows/build.yml

```diff
@@ -26,6 +26,12 @@ jobs:
           pushd literature
           ls ./**/*.yaml | parallel --bar --halt now,fail=1 'svgdigitizer cv --sampling-interval .001 --package --metadata {} {.}.svg --outdir=../data/generated/svgdigitizer/{//}'
           popd
+      - name: copy literature
+        shell: bash -l {0}
+        run: |
+          pushd literature
+          find . -name '*.bib' | cpio -pdm ../data/citations
+          popd
```

Comment:
With this approach, the bib files are copied with the directory tree.
I guess this is not necessary and a list with bib files would be sufficient after all.

I couldn't figure out how to do it.

Reply:
Maybe the bib files should be stored in a `citation` folder not located in `data`, but I think that all data generated for the website should be stored in a single place. Then the folder could be called `resources` with a `data` folder and a `citation` folder.

Reply:
Is a separation necessary?

Reply:
```
data/
     sato_2006_effect_725/
          sato_2006_effect_725.bib
          sato_2006_effect_725_f4a_solid.csv
          sato_2006_effect_725_f4a_solid.json
          sato_2006_effect_725_f4b_solid.csv
          sato_2006_effect_725_f4b_solid.json
```
This structure seems to be ok. What do you think?

Reply:
Yes, you are right. It is better if we put the bib file in the folder with the files. 
In the case that there is at some point another folder in `data` other than `digitized/svgdigitizer` we might have to check for duplicates.

Reply:
See also further discussion on folder structures in #123